components/layout/PageHeader/PageHeader.styles.ts

```diff
@@ -3,10 +3,18 @@ import { basePadding } from "theme/mixins/appearance";
 import { noFlexGapSupport, respond } from "theme/mixins/base";
 import { pxToRem } from "theme/mixins/functions";
 
+import PageHeader from "./PageHeader";
+type Props = React.ComponentProps<typeof PageHeader>;
+
 const MOBILE_BREAK = 40;
 
-export const Header = styled.header`
-  padding-block-start: ${basePadding(6)};
+export const Header = styled.header<Pick<Props, "headerStyle">>`
+  ${({ headerStyle }) =>
+    headerStyle !== "secondary" &&
+    css`
+      padding-block-start: ${basePadding(6)};
+    `}
+
```

Comment:
Here's how I fixed the header padding! First the header prop types needed to be imported (line 6 & 7), then the prop for headerStyle needed to be added to the styled header (line 11). After that, you can use the headerStyle value to add top padding if the style isn't "secondary" ✨